components/reference-data/src/test/java/org/datacleaner/beans/transform/SynonymLookupTransformerTest.java

```diff
@@ -29,13 +29,33 @@
 import org.datacleaner.data.MockInputRow;
 import org.datacleaner.reference.SynonymCatalog;
 import org.datacleaner.reference.TextFileSynonymCatalog;
+import org.junit.Test;
 
-public class SynonymLookupTransformerTest extends TestCase {
+public class SynonymLookupTransformerTest {
 
     private final DataCleanerConfiguration configuration = new DataCleanerConfigurationImpl();
     private final SynonymCatalog sc = new TextFileSynonymCatalog("my synonyms",
             "src/test/resources/synonym-countries.txt", true, "UTF8");
 
+    @Test
+    public void testCaseInsensitiveMathing() throws Exception {
```

Comment:
And, since we've had issues in the case sensitiveness, maybe consider repeating the test without lookupEveryToken